cmd/hiveadmission/main.go

```diff
@@ -10,10 +10,12 @@ import (
 	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
 	hivevalidatingwebhooks "github.com/openshift/hive/pkg/apis/hive/v1/validating-webhooks"
 	"github.com/openshift/hive/pkg/certs"
+	"github.com/openshift/hive/pkg/version"
 )
 
 func main() {
 	log.Info("Starting CRD Validation Webhooks.")
+	log.Infof("Version: %s @ %s", version.String, version.Commit)
```

Comment:
Should this be moved up a line? I would think that the version should be the first thing output.

Reply:
sure, I can do that